components/sysutils/bonnie++/patches/02-cxxflags.patch

```diff
@@ -0,0 +1,14 @@
+--- bonnie++-1.97.3/Makefile.in.orig	2017-10-27 23:45:45.765611579 +0000
++++ bonnie++-1.97.3/Makefile.in	2017-10-27 23:45:53.300765254 +0000
+@@ -9,9 +9,9 @@
+ eprefix=@exec_prefix@
+ #MORE_WARNINGS=-Weffc++
+ WFLAGS=-Wall -W -Wshadow -Wpointer-arith -Wwrite-strings -pedantic -ffor-scope -Wcast-align -Wsign-compare -Wpointer-arith -Wwrite-strings -Wformat-security -Wswitch-enum -Winit-self $(MORE_WARNINGS)
+-CFLAGS=-O2 @debug@ -DNDEBUG $(WFLAGS) $(MORECFLAGS)
++CFLAGS=$(CXXFLAGS) @debug@ -DNDEBUG $(WFLAGS) $(MORECFLAGS)
```

Comment:
Looks strange.. Do the use CFLAGS with CXX?

Reply:
Yeah apparently they missed that the convention is not CFLAGS but CXXFLAGS. But since this is used elsewhere I did not bother replacing it.